tests/acceptance/test_accept_organization_invite.py

```diff
@@ -19,8 +22,19 @@ def setUp(self):
             role='owner',
             teams=[self.team],
         )
-        self.login_as(self.user)
 
-    def test_invite(self):
+    def test_invite_simple(self):
+        self.login_as(self.user)
         self.browser.get(self.member.get_invite_link().split('/', 3)[-1])
         self.browser.snapshot(name='accept organization invite')
+        assert self.browser.element_exists('[data-test-id="join-organization"]')
```

Comment:
I like the use of data-test-id attributes, should we add a method to Browser so they are easier to use? Perhaps something like `browser.element_exists_by_test_id()`?

Reply:
I like that a lot! I notice we also use them quite often with our waits and might be worthwhile to add something similar for this use case as well.